.github/workflows/roman_ci.yml

```diff
@@ -21,55 +21,7 @@ concurrency:
 
 jobs:
   data:
-    name: retrieve current CRDS context, and WebbPSF data
-    runs-on: ubuntu-latest
-    env:
-      OBSERVATORY: roman
-      CRDS_SERVER_URL: https://roman-crds.stsci.edu
-      CRDS_PATH: /tmp/data
-    outputs:
-      data_path: ${{ steps.data.outputs.path }}
-      webbpsf_path: ${{ steps.webbpsf_path.outputs.path }}
-      data_hash: ${{ steps.data_hash.outputs.hash }}
-      crds_path: ${{ steps.crds_path.outputs.path }}
-      crds_context: ${{ steps.crds_context.outputs.pmap }}
-      crds_server: ${{ steps.crds_server.outputs.url }}
-    steps:
-      # crds:
-      - id: crds_context
-        run: >
-          echo "pmap=$(
-          curl -s -X POST -d '{"jsonrpc": "1.0", "method": "get_default_context", "params": ["${{ env.OBSERVATORY }}"], "id": 1}' ${{ env.CRDS_SERVER_URL }}/json/ |
-          python -c "import sys, json; print(json.load(sys.stdin)['result'])"
-          )" >> $GITHUB_OUTPUT
-        # Get default CRDS_CONTEXT without installing crds client
-        # See https://hst-crds.stsci.edu/static/users_guide/web_services.html#generic-request
-      - id: crds_path
-        run: echo "path=${{ env.CRDS_PATH }}" >> $GITHUB_OUTPUT
-      - id: crds_server
-        run: echo "url=${{ env.CRDS_SERVER_URL }}" >> $GITHUB_OUTPUT
-      # webbpsf:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - id: data
-        run: |
-          echo "path=/tmp/data" >> $GITHUB_OUTPUT
-          echo "webbpsf_url=https://stsci.box.com/shared/static/n1fealx9q0m6sdnass6wnyfikvxtc0zz.gz" >> $GITHUB_OUTPUT
-      - run: |
-          mkdir -p tmp/data/
-          mkdir -p ${{ steps.data.outputs.path }}
-      - run: wget ${{ steps.data.outputs.webbpsf_url }} -O tmp/minimal-webbpsf-data.tar.gz
-      - run: tar -xzvf tmp/minimal-webbpsf-data.tar.gz -C tmp/data/
-      - id: data_hash
-        run: echo "hash=${{ hashFiles( 'tmp/data' ) }}" >> $GITHUB_OUTPUT
-      - run: mv tmp/data/* ${{ steps.data.outputs.path }}
-      - uses: actions/cache@v3
-        with:
-          path: ${{ steps.data.outputs.path }}
-          key: data-${{ steps.data_hash.outputs.hash }}-${{ steps.crds_context.outputs.pmap }}
-      - id: webbpsf_path
-        run: echo "path=${{ steps.data.outputs.path }}/webbpsf-data" >> $GITHUB_OUTPUT
+    uses: ./.github/workflows/data.yml
```

Comment:
I think this will attempt to run the `webbpsf-data` job every time, right? Then, if it is skipped, the subsequent jobs that need it will also be skipped

Reply:
oh wait nevermind, I see now the `if: always() ...` line

Reply:
I should add a comment describing that as I'm not sure if that's a common way to do things.

Reply:
Added a comment in 9cdab6831a7e3acc49222a36d3e49d0a74c6bed2